scripts/run_ginkgo.sh

```diff
@@ -3,7 +3,6 @@ set -e
 
 # This script assumes that an AvalancheGo and Subnet-EVM binaries are available in the standard location
 # within the $GOPATH
-# The AvalancheGo and PluginDir paths can be specified via the environment variables used in ./scripts/run.sh.
```

Comment:
Since this comes with new requirements (install kurtosis cli), would it be possible to add or link instructions that are necessary to run the e2e tests with this change?

Reply:
I have added instructions about installing Kurtosis & restarting the engine. Let me know what you think :-)